docs/wiki/GettingStarted.md

```diff
@@ -141,6 +142,10 @@ As the modules we test oftentimes have dependencies to other services, we create
 
 Several fundamental variables are shared among all pipelines and are stored in a pipeline variable file. In case you want to not only leverage the module templates but actually re-use the implemented pipelines & testing framework as well, you need to set up several [variables](./PipelinesDesign#pipeline-variables) & [secrets](./PipelinesDesign#pipeline-secrets) in your environment.
 
+### Name Prefix Token Replacement
+
+Change the default `namePrefix` token value in the [Settings.json](https://github.com/Azure/ResourceModules/blob/main/settings.json) to a 3-5 character string that is used to distinguish your resources names. The default `namePrefix` token has a value `sxx`. Consider this to be one of the first pull requests you need to perform before running any of the repository pipelines. More information on tokens is provided [below](#parameter-file-tokens).
```

Comment:
Can we confirm 5 characters is 'short' enough to not run into resource name limitations?

Reply:
Previously, i used `carml` as a test name in my test environment and did not run into issues back then. However reviewing [Resource name limits](https://docs.microsoft.com/en-us/azure/azure-resource-manager/management/resource-name-rules) documentation. and cross referencing the ones that are risky with our repo, only one concerning is the virtual machine (windows), as it is 15 characters, maybe we can reduce the name [here](https://github.com/Azure/ResourceModules/blob/main/arm/Microsoft.Compute/virtualMachines/.parameters/windows.parameters.json) from 001 --> 1. 5 characters is a good threshold and gives wiggle room for people to make it more unique

Reply:
I wouldn't reduce the vm number suffix that much. I'd either go with 4 chars or 001-->01 (15 chars should be fine and it's now 14)